devimint/src/external.rs

```diff
@@ -77,16 +77,18 @@ impl Bitcoind {
     }
 
     pub async fn mine_blocks(&self, block_num: u64) -> Result<()> {
-        info!(block_num, "Mining bitcoin blocks");
+        info!(?block_num, "Mining bitcoin blocks");
         let client = self.client();
         let addr = client.get_new_address(None, None)?;
         let initial_block_count = client.get_block_count()?;
         client.generate_to_address(block_num, &addr)?;
         while client.get_block_count()? < initial_block_count + block_num {
-            trace!(LOG_DEVIMINT, "Waiting for blocks to be mined");
```

Comment:
Please don't remove `LOG_DEVIMINT`, add the required `taret: ` instead. 

https://github.com/fedimint/fedimint/blob/9a14f7b8379c13546821ae91cf0067e27e0f4426/devimint/src/external.rs#L510

@dpc had a good reason for targets iirc.

Reply:
In devimint it would allow setting one `RUST_LOG` that has filter for devimint and stuff it starts and calls.

Reply:
I took the liberty to resolve this